client/components/QuestionsAnswers/QuestionsAnswers.jsx

```diff
@@ -1,13 +1,264 @@
 import React from 'react';
-import { COLORS } from '../../settings/colors';
 import styled from 'styled-components';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
+import api from '../../api.js';
 
-const QuestionsAnswers = () => (
-  <Header>Q&A Module</Header>
-);
+
+// const AnswerTitle = styled.div`
+//   display: inline-block
+// `
+
+// const AnswerBody = styled.div`
+//   display: inline-block
+// `
+
+const Wrapper = styled.div`
```

Comment:
I believe its the common practice to define these styled components at the bottom of the file -- it'll make it a little bit easier to read as well!